build-tools/src/main/resources/revapi/revapi.json

```diff
@@ -39,5 +41,17 @@
         }
       }
     }
+  },
+  {
+    "extension": "revapi.differences",
+    "id": "differences",
+    "configuration": {
+      "ignore": true
```

Comment:
❓ Should we add a justification to this ignore? (it can be inferred, but might be good to document)

Reply:
If we omit it here, then you have to put it in every other `revapi.json` - but only once, as it's a scalar and will not be merge-able. I figured to reduce confusion, let's just add it in the place where we're least likely to change it, and just never add it anywhere else.